src/sysctl.cr

```diff
@@ -0,0 +1,87 @@
+lib LibC
```

Comment:
This `lib` declaration IMO should be extracted to a separate file for each architecture (`src/lib_c/<arch>/c/sysctl.cr`).